viewer-prototype/src/browser/trace-viewer/components/utils/checkbox-component.tsx

```diff
@@ -4,34 +4,18 @@ type CheckboxProps = {
     id: number
     name: string
     checked: boolean;
-    onChecked: (id: number) => void
+    onChecked: (id: number) => void;
 }
 
-type CheckboxState = {
-    checked: boolean;
-}
-
-export class CheckboxComponent extends React.Component<CheckboxProps, CheckboxState> {
+export class CheckboxComponent extends React.Component<CheckboxProps> {
```

Comment:
If this file is not used anywhere else, you may want to move it in the filter-tree folder